### PR TITLE
prep for v1.2.0 release

### PR DESCRIPTION
**Pull request by jonhealy1**
Minor changes to release v1.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - 
+
+## [v1.2.0] - 2022-04-26
```
Comment (gadomski) [inline CHANGELOG.md:9]: There was no v1.1.3 release, so can you include https://github.com/stac-utils/stac-check/pull/80/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL13 in this header as well?

**Review by gadomski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jonhealy1) [reply CHANGELOG.md:9]: https://pypi.org/project/stac-check/ I released it on pypi but I forgot to make a release on github?

Comment (gadomski) [reply CHANGELOG.md:9]: Oh interesting. Yeah in that case disregard my comment -- can you put a v1.1.3 tag up on github from where that release was cut? Thanks.

Comment (jonhealy1) [reply CHANGELOG.md:9]: I missed this last comment - I will be more diligent next time

Comment (gadomski) [reply CHANGELOG.md:9]: No worries!

**Pull request merged**